src/Symfony/Bridge/PhpUnit/Tests/DnsMockTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use Symfony\Bridge\PhpUnit\DnsMock;
+
+require_once __DIR__.'/../DnsMock.php';
```

Comment:
this is temporary until our phpunit wrapper is updated (PR coming on 2.3 after merge)